gradle-plugins/README.md

```diff
@@ -33,10 +33,8 @@ You can apply the `biz.aQute.bnd.builder` plugin to your project and/or you can
 To make the Bnd Builder Gradle Plugin available to your build, use the following in `settings.gradle`:
 
 ```groovy
-pluginManagement {
```

Comment:
You miss the point of this. It is in settings and thus configures the version of the plugin so that later use in build.gradle files do not need to state the version. This change is wrong.

Reply:
It could be placed either inside `pluginManagement` or not, I'm reverting this change.

Reply:
Got it.